src/containers/Drawer/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { View } from 'react-native';
+import { func } from 'prop-types';
+import { connect } from 'react-redux';
+
+import styles from './styles';
+import { logout } from '../../actions/userActions';
+import Menu from '../../components/menu';
+
+const menuOptions = [{ key: 1, label: 'Map' }, { key: 2, label: 'Chats' }, { key: 3, label: 'Profile' }, { key: 4, label: 'Logout' }];
+
+const Drawer = ({ logout }) => (
+  <View style={styles.container}>
+    <Menu
+      optionsList={menuOptions}
+      onOptionSelected={(key) => {
+        switch (key) {
+          case 1:
+          case 2:
+          case 3:
```

Comment:
Why do you have this empty cases?

Reply:
Ok, this switch is needed for next pr's, each case indicates an option from sliding menu was selected. Maybe i can create some constants indicating the option to avoid this magic numbers.